Utilities/PPS/bin/PPSOpticsCalibration.cc

```diff
@@ -0,0 +1,591 @@
+#include "Utilities/PPS/interface/PPSOpticsCalibrator.h"
```

Comment:
Maybe this file should be called ```PPSOpticsCalibrator.cc```, instead?
Out of curiosity: why keeping the implementation file in the /bin subdirectory?

Reply:
It is part of the ongoing PR #24490